etcdctl/ctlv3/command/status_command.go

```diff
@@ -0,0 +1,49 @@
+// Copyright 2016 CoreOS, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package command
+
+import (
+	"fmt"
+	"os"
+
+	v3 "github.com/coreos/etcd/clientv3"
+	"github.com/spf13/cobra"
+)
+
+// NewStatusCommand returns the cobra command for "Status".
+func NewStatusCommand() *cobra.Command {
+	return &cobra.Command{
+		Use:   "status",
+		Short: "status statusments the storage of the etcd members with given endpoints.",
+		Run:   statusCommandFunc,
+	}
+}
+
+func statusCommandFunc(cmd *cobra.Command, args []string) {
+	c := mustClientFromCmd(cmd)
+
+	statusList := []*v3.StatusResponse{}
+	for _, ep := range c.Endpoints() {
+		ctx, cancel := commandCtx(cmd)
+		resp, err := c.Status(ctx, ep)
+		cancel()
+		if err != nil {
+			fmt.Fprintf(os.Stderr, "Failed to get the status of etcd member[%s] (%v)\n", ep, err)
+		} else {
+			statusList = append(statusList, resp)
+		}
+	}
+	display.MemberStatus(statusList)
+}
```

Comment:
> we should exit with non-zero if there is an error

For example we have 3 endpoints, if we failed to get the status of ANY endpoint, we should exit with non-zero? How about: just print out the err information, and print the status information when ` len(statusList) != 0`